hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -3113,27 +3120,32 @@ private DBCheckpoint getDBCheckpointFromLeader(String leaderId) {
     return null;
   }
 
+  void stopServices() throws Exception {
+    keyManager.stop();
+    stopSecretManager();
+    metadataManager.stop();
+
+    // s3SecretManager should also be stopped
```

Comment:
Right now, we don't have any stop() for S3SecretManager and PrefixManager.
And also for these, only read/write operations will happen. Write will go through double buffer and read's will not happen anyway as this is not leader. Do we still see issue?

Reply:
If writes can happen then we should stop it, right?

Reply:
Thanks for confirming.
Then we can remove the comments.